### PR TITLE
Updated The billing logic to handle Unmanaged VM's and lsblk

### PR DESCRIPTION
**Pull request by arisettisanjana**
Had modified the lsblk  --json command as --json option is not available in all distros.
Also included a condition that helps in handling the case for unmanaged VM's.

### CODE REVIEW

File: VMBackup/main/Utils/SizeCalculation.py [hunk 1]
```diff
@@ -60,17 +60,24 @@ def __init__(self,patching, hutil, logger,para_parser):
             self.output_lsscsi = ""
             self.lsscsi_list = []
         try:
-             self.output_lsblk = json.loads(os.popen("lsblk --json --output name,mountpoint").read())
+            self.output_lsblk = os.popen("lsblk -n --list --output NAME,MOUNTPOINT").read().strip().split("\n")
```
Comment (iamashwani-who) [inline VMBackup/main/Utils/SizeCalculation.py:63]: Any specific reason for casing changes from name,mountpoint to NAME,MOUNTPOINT?

File: VMBackup/main/Utils/SizeCalculation.py [hunk 2]
```diff
@@ -137,38 +144,47 @@ def device_list_for_billing(self):
...
+                sdf
+                sdg
+        '''
+        for item in self.output_lsblk:
+            item_split = item.split()
+            if(len(item_split)==2):
+                device = item_split[0]
+                mount_point = item_split[1]
+            else:
+                mount_point = ""
```
Comment (iamashwani-who) [inline VMBackup/main/Utils/SizeCalculation.py:179]: can we log a statement that that item split did not return 2 items and hence skipping?

File: VMBackup/main/Utils/SizeCalculation.py [hunk 3]
```diff
@@ -137,38 +144,47 @@ def device_list_for_billing(self):
...
+                sdc1          /mnt/sdc1
+                sdc2          /mnt/sdc2
+                sdc3
+                sde
+                sde1          /mnt/sde1
+                sdf
+                sdg
+        '''
+        for item in self.output_lsblk:
+            item_split = item.split()
```
Comment (iamashwani-who) [inline VMBackup/main/Utils/SizeCalculation.py:174]: Should be reset the values of mount_point and device at the top of the loop?

File: VMBackup/main/Utils/SizeCalculation.py (hunk 2 above)
Comment (arisettisanjana) [reply VMBackup/main/Utils/SizeCalculation.py:179]: might not need it.

**Review by iamashwani-who — APPROVED** [review]

**Review by rakrashba — CHANGES_REQUESTED** [review]

**Pull request closed without merging**